js/gratipay/dropdown.js

```diff
@@ -0,0 +1,117 @@
+Gratipay.Dropdown = function(selector) {
+    var $ul = $(selector);
+    var $labels = $('label', $ul);
+
+    // state for vertical position
+    var topFactor = 0;      // float between 0 and $labels.length-1
+    var maxTopFactor = $labels.length - 1;
```

Comment:
I believe that's prevented by the condition in the template at https://github.com/gratipay/gratipay.com/blob/371e17a8e089515cb4234e9b7c361de95df02fa1/www/on/npm/%25package/index.html.spt#L61, no? If there are no emails then there is no `.gratipay-dropdown` rendered and therefore we never get here.